src/main/java/ZadaniaWzorceProjektowe_10_09/Zadanie3/CompositeEmployee.java

```diff
@@ -0,0 +1,39 @@
+package ZadaniaWzorceProjektowe_10_09.Zadanie3;
+
+import java.util.ArrayList;
+
+public class CompositeEmployee extends Employee {
+    private ArrayList<Employee> employees = new ArrayList<Employee>();
```

Comment:
Może lista powinna być finalna? Generalnie ważne jest by oddzielać zmienne tzw. immutable i muttable: https://www.quora.com/What-is-the-difference-between-mutable-and-immutable-objects-in-Java